demisto_sdk/commands/validate/file_validator.py

```diff
@@ -650,6 +651,24 @@ def validate_all_files(self):
         for file in all_files_to_validate:
             self.run_all_validations_on_file(file, file_type=find_type(file))
 
+    def validate_pack(self):
+        """Validate files in a specified pack"""
+        packs = glob(f'{PACKS_DIR}/*')
+        if self.pack_path not in packs:
+            raise Exception(F'File {self.pack_path} was not found\nMake sure you are using relative path')
+        pack_files = {file for file in glob(fr'{self.pack_path}/**', recursive=True) if not os.path.isdir(file)}
+        pack_unique_files_validator = PackUniqueFilesValidator(os.path.abspath(self.pack_path))
+        pack_errors = pack_unique_files_validator.validate_pack_unique_files()
+        if pack_errors:
+            print_error(pack_errors)
+            self._is_valid = False
+        for file in pack_files:
+            if file.endswith('pack_metadata.json'):
```

Comment:
this does not need to be validated because it is validated in `validate_pack_unique_files` ? what about the rest of the files validated there? why are we not skipping them ?

Reply:
No need. The `run_all_validations_on_file` is not checking anything that is not "yml", "md" "json". so secrets and pack ignore skipped in the check anyways

Reply:
And yes, all pack unique files, are validated in `validate_pack_unique_file`

Reply:
ok lets add a comment here for that